### PR TITLE
Fixed the indentation of "All Posts" cards.

### PR DESCRIPTION
**Pull request by anishsoni29**
## Summary

Fixing the indentation of the All Posts with the image box.
Screenshot 2023-12-15 at 3 18 05 AM

## Description

Fixed the indentation gap by removing the extra margin and adding the gap in the "All Posts" section.

## Issue(s) Addressed

#81

### CODE REVIEW

**Review by niikkhilsharma — APPROVED** [review]

**Review by niikkhilsharma — APPROVED** [review]
This code works fine.

**Comment by krishnaacharyaa** [comment]
Thank you @anishsoni29 for the quick PR, but there seems to be still some changes required, here we covered only the shimmers not the actual UI
image

Thank you @niikkhilsharma for taking out your time for reviewing, couple of inputs 
1. The code actually doesn't look fine in all the scenario, so kindly be very sure, when you comment.
2. And it's not about "it works fine", it is the bare minimum, but the things to focus on is quality of code, is minimal change carried out, was all the changes actually required? That comes to picture.

> Co-authoured what's about this in the commit, I didn't get the point, was there any reason behind?

**Comment by niikkhilsharma** [comment]
I really appreciate your feedback. I'll try to check for more scenarios in which the code would fail.
Thanks you 😊

File: frontend/src/pages/home-page.tsx [hunk 1]
```diff
@@ -52,8 +52,8 @@ function HomePage() {
       </div>
       <div className="mx-4 md:mx-8 lg:mx-16">
         <BlogFeed />
-        <h1 className="text-2xl font-semibold dark:text-dark-primary">All Posts</h1>
-        <div className="flex flex-wrap">
+        <h1 className="mb-4 text-2xl font-semibold dark:text-dark-primary">All Posts</h1>
+        <div className="flex flex-wrap gap-4">
           {posts.length === 0
```
Comment (krishnaacharyaa) [inline frontend/src/pages/home-page.tsx:57]: Why gap-4 is added? It is breaking the UI by pushing the posts to below at the right we have empty white space now
image

And not exactly sure why mb-4 is added here, i understand to compensate the m-4 removal in the post-card-skeleton, but then there would be other places also to be considered

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Kindly address the changes, and follow the contribution guidelines 5th point staging commit

**Comment by krishnaacharyaa** [comment]
#86 Fixes this issue

**Pull request closed without merging**